hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -456,4 +456,15 @@ private Constants() {
   public static final String FAIL_INJECT_INCONSISTENCY_PROBABILITY =
       "fs.s3a.failinject.inconsistency.probability";
 
+  public static final String MULTIPART_DOWNLOAD_PART_SIZE = "fs.s3a.multipartdownload.part-size";
+  public static final long DEFAULT_MULTIPART_DOWNLOAD_PART_SIZE = 8_000_000;
+
+  public static final String MULTIPART_DOWNLOAD_CHUNK_SIZE = "fs.s3a.multipartdownload.chunk-size";
+  public static final long DEFAULT_MULTIPART_DOWNLOAD_CHUNK_SIZE = 262_144;
+
+  public static final String MULTIPART_DOWNLOAD_BUFFER_SIZE = "fs.s3a.multipartdownload.buffer-size";
```

Comment:
Unit?

Reply:
I'm following the convention here: https://hadoop.apache.org/docs/r2.8.0/hadoop-project-dist/hadoop-common/core-default.xml: fs.s3a.multipart.threshold